orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationModuleConfig.java

```diff
@@ -28,7 +29,6 @@
  * </ul>
  */
 @Value
-@With
```

Comment:
Why remove the option to set `LLMModuleConfig` manually on this API?

Reply:
re-added

Reply:
re-added